content/F2015/F15-event-dropbox-coffee.md

```diff
@@ -0,0 +1,17 @@
+title: Dropbox Coffee Chats
+<<<<<<< HEAD
```

Comment:
Uh oh, somehow this got left over from a merge conflict? I'm still not understanding when this merge came up :S